<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArgDialog</class>
 <widget class="QDialog" name="ArgDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>favicon.ico</normaloff>favicon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(0, 43, 54);</string>
  </property>
  <widget class="QDialogButtonBox" name="argCommandButtons">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>510</y>
     <width>211</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 85, 0);
font: 87 11pt &quot;Noto Sans&quot;;
color: rgb(0, 43, 54);
font-weight: 700;
padding: 4px;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="addArgument">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>190</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 85, 0);
font: 87 11pt &quot;Noto Sans&quot;;
color: rgb(0, 43, 54);
font-weight: 700;
padding: 4px;</string>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="argDialogTitle">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 255);</string>
   </property>
   <property name="text">
    <string>Add Arguments</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addFile">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 85, 0);
font: 87 11pt &quot;Noto Sans&quot;;
color: rgb(0, 43, 54);
font-weight: 700;
padding: 4px;</string>
   </property>
   <property name="text">
    <string>File</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Noto Sans&quot;;
color: rgb(131, 148, 150);
font-weight: 700;</string>
   </property>
   <property name="text">
    <string>Items to send</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="argList">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>481</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 11pt &quot;Noto Sans&quot;;
background-color: rgb(7, 54, 66);
color: rgb(131, 148, 150);
font-weight: 700;
</string>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>100</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>200</number>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>100</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>100</number>
   </attribute>
   <column/>
   <column/>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>58</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Noto Sans&quot;;
color: rgb(131, 148, 150);
font-weight: 700;</string>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Noto Sans&quot;;
color: rgb(131, 148, 150);
font-weight: 700;</string>
   </property>
   <property name="text">
    <string>Input</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Noto Sans&quot;;
color: rgb(131, 148, 150);
font-weight: 700;</string>
   </property>
   <property name="text">
    <string>Attachment</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="argInput">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 11pt &quot;Noto Sans&quot;;
background-color: rgb(7, 54, 66);
color: rgb(131, 148, 150);
font-weight: 700;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Noto Sans&quot;;
color: rgb(131, 148, 150);
font-weight: 700;</string>
   </property>
   <property name="text">
    <string>Time</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTime">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>194</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="argType">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 11pt &quot;Noto Sans&quot;;
background-color: rgb(7, 54, 66);
color: rgb(131, 148, 150);
font-weight: 700;
</string>
   </property>
   <item>
    <property name="text">
     <string>description</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>hashtag</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>requested by</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>promote/share</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>link/bio</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="devTestButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>590</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(130, 255, 121);
background-color: rgb(2, 180, 43);
font: 87 11pt &quot;Noto Sans&quot;;
color: rgb(0, 0, 0);
font-weight: 700;
padding: 4px;</string>
   </property>
   <property name="text">
    <string>TEST</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>argCommandButtons</sender>
   <signal>accepted()</signal>
   <receiver>ArgDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>argCommandButtons</sender>
   <signal>rejected()</signal>
   <receiver>ArgDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
